src/tokenizer.rs

```diff
@@ -3543,4 +3546,17 @@ mod tests {
         ];
         compare(expected, tokens);
     }
+
+    #[test]
+    fn test_national_strings() {
+        let dialect = PostgreSqlDialect {};
```

Comment:
could we test the behavior both for dialects that support and don't support backslash? (We can probably reuse the `all_dialects_where` helper function).
Also maybe the test could be called `test_national_strings_backslash_escape` or similarly specific to the test is covering?

Reply:
Great suggestion! How does it look now?